src/services/API/Session.ts

```diff
@@ -0,0 +1,44 @@
+import { apiClient } from "./RootClient";
+import { sessionsEndpoint, sessionAnswersEndpoint } from "./Endpoints";
+import {
+  SessionAPIResponse,
+  SessionAnswerAPIResponse,
+  submittedAnswer,
+} from "../../types";
+
+export default {
+  /**
+   * returns the details for a quiz
+   * @param {string} quizId - id of the quiz for which the session is to be created
+   * @param {string} userId - id of the user for which the session is to be created
+   * @returns {Promise<SessionAPIResponse>} data corresponding to the session
+   */
+  async createSession(
```

Comment:
To maintain consistency across our "products", call this component `UserQuizSession`?

Reply:
Even within the quiz repo? It always includes the Quiz part implicitly no?